lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -96,9 +93,12 @@ def rabbitmq_settings
 
         s[:timeout] = @connection_timeout || 0
         s[:heartbeat] = @heartbeat || 0
-        s[:tls] = @ssl if @ssl
-        s[:tls_certificate_path] = @tls_certificate_path || ""
-        s[:tls_certificate_password] = @tls_certificate_password || ""
+
+        # This MUST be passed as a string or the cert options will be ignored
+        s[:tls] = @ssl.is_a?(String) ? @ssl : "TLSv1.2"
```

Comment:
Ahha, ok, so we can still :validate an array, and TLS v1.3 is a long way away, I think. ;P

For the default, instead of doing `@ssl.is_a?(String)` you can just do use the truthy value of @ssl, since the default is nil, and nil is falsey. Alternate question, when would @ssl be not a string?